src/api/handlers/guild.ts

```diff
@@ -206,6 +207,32 @@ export function swapChannels(
   );
 }
 
+/** Edit the channel permission overwrites for a user or role in this channel. Requires MANAGE_ROLES permission. */
+export async function editChannelOverwrite(
+  guildID: string,
+  channelID: string,
+  options: Overwrite,
+) {
+  const hasPerm = await botHasPermission(
+    guildID,
+    ["MANAGE_ROLES"],
+  );
+  if (!hasPerm) {
+    throw new Error(Errors.MISSING_MANAGE_ROLES);
+  }
+
+  const body = {
+    allow: calculateBits(options.allow),
+    deny: calculateBits(options.deny),
+    type: options.type,
+  };
```

Comment:
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/34645569/105061668-33c25c00-5a93-11eb-9d03-a979f32d6f31.png)
You can commit the suggestion using the "Commit suggestion" button :)